Entwurf/Zuarbeit Entwurf.tex

```diff
@@ -0,0 +1,80 @@
+\documentclass[a4paper,10pt]{article}
+\usepackage[utf8]{inputenc}
+\usepackage{tocbibind}
+\usepackage{xcolor}
+\usepackage{tikz}
+\usepackage{hyperref}
+\usepackage[ngerman]{babel}
+\usepackage{graphicx}
+\graphicspath{ {./exports/png/} }
+
+
+\begin{document}
+\section{Grobentwurf}
+Der Grobentwurf wurde so gestaltet, dass er nach der 3-Schichtenarchitektur aufgebaut ist. Genauer gesagt wurde innerhalb dieser, dass Model-View-Controller Muster verwendet. 
+Dabei wurde die View und der Controller für die Präsentationsschicht und das Model für die Logikschicht verwendet.Die Datenhaltungsschicht der 3 Schichtenarchitektur entspricht unserem Repository
+
+\subsection{Einzelne Elemente des MVC}
+  \subsubsection{Model} 
+    Das Model erhält die Domänendaten.Es wird durch seine Funktionalitäten verändert.
+   \subsubsection{View}
+   Die View ist für die Darstellung die die Interaktion mit dem Nutzer zuständig, solbald der Nutzer zum Beispiel auf der Oberfläche einen Klick ausführt, wird dies sofort an den Controller weitergegeben, der wiederrum das Model kontaktiert.
+   \subsubsection{Controller}
+   Der Controller ist ein zentrales Element, dass die Daten der View entgegenimmt und dann ein Event behandelt, die vom Controller ausgelöst werden.
+   \subsection{Paketdiagramm des Grobentwurfs}
+   \begin{figure}
+    \includegraphics[scale=0.5]{Entwurf_PD_Grobentwurf.png}
+    \label{fig:Paketdiagramm_Grobentwurf}
+    \caption{Paketdiagramm Grobentwurf}
+   \end{figure} 
+
+   \subsection{Erläuterungen zum Paketdiagramm}
```

Comment:
Die obigen Texte sind ja quasi die Erläuterung. Diese subsection kann weg, dafür kannst du in den Texten oben mehr direkten Bezug auf das PD nehmen